From 41e65c1c118d694529799e52b87073ed28ceb313 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Wed, 25 Oct 2023 22:44:24 -0700
Subject: [PATCH] Don't really install this test

---
 libtaskmanager/autotests/CMakeLists.txt | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/libtaskmanager/autotests/CMakeLists.txt b/libtaskmanager/autotests/CMakeLists.txt
index 9600e4f..7f59a58 100644
--- a/libtaskmanager/autotests/CMakeLists.txt
+++ b/libtaskmanager/autotests/CMakeLists.txt
@@ -26,12 +26,6 @@ if(NOT CMAKE_SYSTEM_NAME MATCHES "Linux")
 endif()
 add_executable(waylandtasksmodeltest waylandtasksmodeltest.cpp)
 target_link_libraries(waylandtasksmodeltest PRIVATE taskmanager Qt::Test Qt::GuiPrivate Qt::WaylandClient Wayland::Client KF5::Service KF5::IconThemes KF5::WindowSystem)
-# Do not really install the test
-if(DEFINED ENV{KDECI_BUILD})
-install(TARGETS waylandtasksmodeltest ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
-else()
-install(TARGETS waylandtasksmodeltest DESTINATION ${CMAKE_BINARY_DIR})
-endif()
 
 add_test(
     NAME waylandtasksmodeltest
-- 
2.42.0

